metal/resource_metal_project_test.go

```diff
@@ -88,6 +88,11 @@ func TestAccMetalProject_BGPBasic(t *testing.T) {
 						"2SFsdfsg43"),
 				),
 			},
+			{
```

Comment:
If you add the import check to the project tests, why not adding it to the `TestAccMetalProject_Basic` too?

Reply:
I did at first and then noticed `TestAccMetalProject_importBasic` which does the same.